src/screens/HyperSwitch/Utils/CountryUtils.res

```diff
@@ -751,259 +751,6 @@ type countiesCode =
   | ZM
   | ZW
 
```

Comment:
Hi, @shubham0804 can you please add back the CountryUtils file back we probably need it. The rest looks fine to me. Don't change here it's okay. It will be used eventually.